net/tstun/wrap.go

```diff
@@ -615,16 +668,24 @@ func (c *natV4Config) selectSrcIP(oldSrc, dst netip.Addr) netip.Addr {
 	return oldSrc
 }
 
-// natV4ConfigFromWGConfig generates a natV4Config from nm.
-// If v4 NAT is not required, it returns nil.
-func natV4ConfigFromWGConfig(wcfg *wgcfg.Config) *natV4Config {
+// natConfigFromWGConfig generates a natFamilyConfig from nm,
+// for the indicated address family.
+// If NAT is not required for that address family, it returns nil.
+func natConfigFromWGConfig(wcfg *wgcfg.Config, v6 bool) *natFamilyConfig {
```

Comment:
do you have to run this loop twice? can we have this create two different configs and return them? like

```
func natConfigFromWGConfig(wcfg *wgcfg.Config) (v4, v6 *natFamilyConfig) {
```

Reply:
At that stage we might as well just return `*natConfig`. I'll do that

Reply:
Done, but honestly the duplication is jarring and i think i prefer the loop. Can we reconsider?